README.md

```diff
@@ -208,6 +208,16 @@ acr purge \
     --repository-page-size 10
 ```
 
+#### Include-locked flag

```

Comment:
Can we add a warning to this? Maybe a link to https://learn.microsoft.com/en-us/azure/container-registry/container-registry-image-lock to note that this is the lock types that are being referenced?